owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java

```diff
@@ -753,12 +756,14 @@ private void initTargetFolder() {
     public boolean onCreateOptionsMenu(Menu menu) {
         MenuInflater inflater = getMenuInflater();
         inflater.inflate(R.menu.main_menu, menu);
-
-        mSearchView = (SearchView) menu.findItem(R.id.action_search).getActionView();
+        MenuItem menuItem = menu.findItem(R.id.action_search);
```

Comment:
Cool! You can do the same with `mSearchView` 😀

Reply:
Perfect. It's done ✅